.github/workflows/macos-ci-x86_64.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
           # *DH
 
           # Set compiler and MPI
-          spack config add "packages:all:providers:mpi:[openmpi@4.1.4]"
+          spack config add "packages:all:providers:mpi:[openmpi]"
```

Comment:
This was causing the CI to fail because the most recent version is 4.1.5. If we want to include the version number, then it needs to be updated, but it works without. What's your preference?

Reply:
I would add 4.1.5 just to be consistent, but I'll leave it up to you.